src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1765,6 +1768,17 @@ static bool processOnChangeConditionForUpdateContext
       }
     }
   }
+  if (covered == true)
+  {
+    for (unsigned int jx = 0; jx < attrL.size(); jx++)
+    {
+      const std::string& _type = "NONE";
```

Comment:
Actually, `"None"` should be used. Better use constant DEFAULT_ATTR_NULL_TYPE, i.e:

```suggestion
      const std::string& _type = DEFAULT_ATTR_NULL_TYPE;
```

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/4068/commits/05ce98dfdba3942ab7a41a5bf76b2903ddeb1e72